### ISSUE TITLE
Form submission

### ISSUE THREAD

**Issue by nafisa20** [fac21/Final-project-NSMM#53]
The submit button doesn't look like it works as the same page still appears and when we checked the code we noticed the button wasn't linked to another page

**Comment by jamdelion** [comment]
Yeah when you sign up, it's not obvious that it has worked - you need to reload the page and then you're logged in. Maybe put a comment saying "reload this window to finish signing in"
